narwhals/series.py

```diff
@@ -111,6 +111,32 @@ def min(self) -> Any:
         return self._series.min()
 
     def max(self) -> Any:
+        """
+        Get the maximum value in this Series.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s = [1, 2, 3]
+            >>> s_pd = pd.Series(s)
+            >>> s_pl = pl.Series(s)
+
+            We define a data-frame agnostic function:
+
+            >>> def func(s_any):
+            ...     s = nw.from_native(s_any, series_only=True)
+            ...     s = s.max()
+            ...     return nw.to_native(s)
+
+            We can then pass either pandas or Polars to `func`:
+
+            >>> func(s_pd) # doctest:+SKIP
```

Comment:
why do we need to skip this?

Reply:
actually, `s.max()` should just return a scalar

So for this one, you can skip the `nw.to_native(s)` part and just do
```
return s.max()
```